env.local.platform.zsh

```diff
@@ -2,24 +2,18 @@
 
 source _env.check-docker.zsh || return 1
 
+
+
 name=$(basename $0); name=$name:r:s/env.//
 echo $name
 source set-prompt-for-env.zsh $name 
 
 # Build ####
-export JAVA_HOME=$(/usr/libexec/java_home -v11)
+export JAVA_HOME=$(/usr/libexec/java_home -v21)
 export DPM_REPO=$HOME/src/streamsets/domainserver-master
 
 # Auth ####
-export ONE_PASSWORD_ITEM=local.platform
-export ASTER_LOGIN_URL=http://host.docker.internal:4200
-export ASTER_SCH_URL=http://host.docker.internal:18631
-export ASTER_EMAIL=$(op read "op://Employee/$ONE_PASSWORD_ITEM/username")
-export ASTER_EMAIL_PWD=$(op read "op://Employee/$ONE_PASSWORD_ITEM/password")
-export ASTER_URL=${ASTER_LOGIN_URL}
-export ASTER_USER_EMAIL=${ASTER_EMAIL}
-export ASTER_USER_PASSWORD=${ASTER_EMAIL_PWD}
-export FIREBASE_API_KEY=$(op read "op://Employee/$ONE_PASSWORD_ITEM/firebase-api-key") 
+source _env.platform.zsh
```

Comment:
There is a misunderstanding on the original purpose of `_env.platform.zsh`. Maybe it should have been named `_env.cloud-platform.zsh`: it was only intended to be called from https://github.com/xverges/streamsets-dev-envs/blob/main/env.dev.my-org.zsh or any other future scripts setting credentials for specific cloud instances+orgs.

Thus, the definitions to start SDCs or the example to start STF tests are important to keep, as is the ability to work with another instance/organization with a simple modification to `env.dev.my-org.zsh`.

That said, I am happy to see duplication go away!

Reply:
Moved getting the credential to operate file and reverted the SDCs and STF

Reply:
There is some duplication issue that I have not debugged: when I run `source env.local.platform.zsh`, I get

```
local.platform
Default value will be used as ONE_PASSWORD_ITEM=local.platform
Auth set from 1Password
local.platform
---
<$HELP> is echoed twice
```